components/tenant-fetcher/internal/tenant/service.go

```diff
@@ -62,10 +62,14 @@ func (s *service) Create(writer http.ResponseWriter, request *http.Request) {
 		http.Error(writer, fmt.Sprintf("Property %q not found in body or it is not of String type", s.config.TenantProviderTenantIdProperty), http.StatusInternalServerError)
 		return
 	}
+	customerId := gjson.GetBytes(body, s.config.TenantProviderCustomerIdProperty)
+	subdomain := gjson.GetBytes(body, s.config.TenantProviderSubdomainProperty)
```

Comment:
Subdomain is missing is CIS1 case and you may log here the for corresponding tenant the subdomain was not found.